corehq/apps/ota/const.py

```diff
@@ -0,0 +1 @@
+DEVICES_PER_USER = 100
```

Comment:
How about making this a setting to that it can be changed without a code deploy and 3rd parties can configure their own limits.

On the topic of 3rd parties, this change probably warrants a changelog entry.

Reply:
Yeah we had talked about potentially creating a new SQL model to update values like this quickly, but settings seems perfectly fine for now, and definitely better than a constant. Moved to settings in 93e48ec2ed1efb96e5cfa3a7591396b8ae9811e6.

Reply:
And agreed on the changelog entry!